src/Sylius/Bundle/PromotionBundle/Resources/config/services/console_command.xml

```diff
@@ -5,7 +5,7 @@
         <service id="Sylius\Bundle\PromotionBundle\Console\Command\GenerateCouponsCommand">
             <argument type="service" id="sylius.repository.promotion" />
             <argument type="service" id="sylius.promotion_coupon_generator" />
-            <tag name="console.command" command="sylius:promotion:generate-coupons"/>
```

Comment:
indeed, you're right, I think I forgot this description removal on previous console commands.